lightkurve/targetpixelfile.py

```diff
@@ -734,8 +734,11 @@ def plot(self, ax=None, frame=0, cadenceno=None, bkg=False, aperture_mask=None,
         cadenceno : int, optional
             Alternatively, a cadence number can be provided.
             This argument has priority over frame number.
-        bkg : bool
+        bkg: bool
             If True, background will be added to the pixel values.
+        column :
```

Comment:
Make sure we specify the expected data type:
```suggestion
        column : str
```